demisto_sdk/commands/common/tools.py

```diff
@@ -767,7 +767,7 @@ def get_file(file_path: Union[str, Path], type_of_file: str, clear_cache: bool =
     if clear_cache:
         get_file.cache_clear()
 
-    file_path = Path(file_path).absolute()
+    file_path = Path(get_content_path()).joinpath(file_path)  # type: ignore
```

Comment:
i guess mypy screams because of the optional `None` output of `get_content_path()`, please either fix it there (perhaps use `or ""`?), or use a specific ignore (rather than a catch-all `type: ignore`)